### PR TITLE
chore: v0.11.1 release links in README

### LINKED ISSUE #1415: Release 0.11.0
# Release Notes

- Web UI: [`v2.8.0`](https://github.com/ipfs-shipyard/ipfs-webui/releases/v2.8.0)
- IPFS: [`go-ipfs v0.5.0-rc4`](https://github.com/ipfs/go-ipfs/releases/tag/v0.5.0-rc4)

### 🚨 Breaking Changes

- fix: remove old auto launch method (#1416)
- fix: ipfs daemon flags (#1437)

### ✨  Features

- feat: support for go-ipfs 0.5 (#1392) 
- feat: run garbage collector (#1407)
- feat: update ipfsd-ctl to 4.x (#1411)
- feat: improved update experience (#1414)
- feat(ux): improve download cid (#1419)
- feat(ux): improve gc (#1420)
- feat(ux): better error messages (#1421)
- feat(ux): separate open from other actions (#1424)
- feat(ux): move preferences to menubar (#1425) 
- feat: allow custom ipfs binary (#1427)
- feat(ux): open web ui at login (#1429)
- feat: add "Help Translation" under "About" (#1447)
- feat: go-ipfs 0.5 (https://github.com/ipfs-shipyard/ipfs-desktop/pull/1463)

### 🛠 Fixes and Maintenance

- fix: drag&drop files and folders (https://github.com/ipfs-shipyard/ipfs-desktop/issues/1287#issuecomment-620604299)
- fix: start on login (#1334)
- fix: notify when new update is available (#1384)
- chore: update to electron 8.x (#1404)
- fix: write path when it does not exist (#1405)
- refactor: remove babel (#1406)
- fix: circular dependency between dialogs and daemon (#1408)
- fix: use portfinder instead of get-port (#1410)
- fix: screenshot taking (#1418)
- fix: disable updates in development (#1423)
- docs: clarify protocol usage (#1426)
- fix: translate 'Select Directory' (#1428)
- refactor: move utilities to utils directory (#1434)
- fix: repo fsck is deprecated (#1438)
- refactor: use store migrations and defaults (#1443)
- fix(ux): show dock when prompt is visible (#1450)
- fix: keep track of last opened page (#1452)

# Before Release

- [x] Make sure i18n is updated (`tx pull -a`)
- [x] Make sure we're using the latest released version of Web UI
- [x] Make sure we're using the latest released version of `go-ipfs`
- [x] Update Release Notes accordingly

# Release

- [x] Release Candidate
	- [x] 💻 Bump version on `package.json`
	- [x] 💻 `git tag $version`
	- [x] 💻 `git push && git push --tags`
	- [x] ⏰ Wait...
	- [x] 🔗 Link release notes to this issue
- [ ] Final
	- [x] 📝 Create draft and copy the release notes above
	- [x] 💻 Bump version on `package.json`
	- [x] 💻 `git tag $version`
	- [x] 💻 `git push && git push --tags`
	- [x] ⏰ Wait...
	- [x] 📃 Copy the release notes above to the GitHub release
    - [ ] :police_officer:  Notarize `.dmg` at Apple (context: [#1365](https://github.com/ipfs-shipyard/ipfs-desktop/issues/1211))
          1. Download `.dmg` from `https://github.com/ipfs-shipyard/ipfs-desktop/releases/vA.B.C`
          2. Ensure `APPLEID` and `APPLEIDPASS` are set either as environment variables or entries in `.env` file. Those need to belong to the same org as cert used for signing.
         3. Run `node pkgs/macos/notarize-cli.js ./IPFS-Desktop-A.B.C.dmg`
	- [ ] 🚢 Ship!
	- [ ] 📑 Update README.md versions
	- [ ] 📦 Update Homebrew
	- [ ] 📦 Update Chocolatey

# Postponed

_If any..._

### PR DESCRIPTION
**Pull request by lidel**
This PR updates links and badges with stats to point at 0.11.0 release.

This should be merged when we publish release and close #1415

### CODE REVIEW

**Comment by rafaelramalho19** [comment]
Shouldn't we have a "latest" version download so we don't need to update these links with every deploy?

**Comment by lidel** [comment]
@rafaelramalho19 that would be great, but afaik it is not possible because platform-specific packages include version number in their filenames.

If you see a way to remove this manual step while keeping some visual stats  on which package type is more popular, I am all for it, simplifying release process would be great.

**Comment by rafaelramalho19** [comment]
I'll look into it later when we need future releases 😄

File: README.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ You can find releases notes and older versions on the [releases](https://github.
 
 ### Install from Source
 
-To install it from source you need [Node.js](https://nodejs.org/en/) `>=10.4.0` and
+To install it from source you need [Node.js](https://nodejs.org/en/) `>=12` and
 need [npm](npmjs.org) `>=6.1.0` installed. This uses [`node-gyp`](https://github.com/nodejs/node-gyp) so **you must take a look** at their [platform specific dependencies](https://github.com/nodejs/node-gyp#installation).
 
 Then the follow the steps below to clone the source code, install the dependencies and run it the app:
```
Comment (jessicaschilling) [inline README.md:103]: “Then the follow” -> “Then follow”

File: README.md [hunk 2]
```diff
@@ -128,6 +128,7 @@ Other languages are periodically pulled from [Transifex](https://www.transifex.c
     1. Download `.dmg` from `https://github.com/ipfs-shipyard/ipfs-desktop/releases/vA.B.C`
     2. Ensure `APPLEID` and `APPLEIDPASS` are set either as environment variables or entries in `.env` file. Those need to belong to the same org as cert used for signing.
     3. Run `node pkgs/macos/notarize-cli.js ./IPFS-Desktop-A.B.C.dmg`
+- Update links and badges in `README` to point at new version (`A.B.C`)
```
Comment (rafaelramalho19) [inline README.md:131]:
```suggestion
- Update links and badges in `README` to point to the new version (`A.B.C`)
```

**Pull request merged**